### ISSUE TITLE
与kanban组件冲突

### ISSUE THREAD

**Issue by dongjinl** [Quorafind/Obsidian-Templify#11]
打开该插件后，看板创建的文件，添加卡片会出现双行，且还有其他BUG，当第二次打开kanban创建的任一文件，都显示一篇空白
![obsidian看板BUG](https://github.com/user-attachments/assets/e5b01612-b9fb-4cbe-b3bd-f699e5ec64d3)

**Comment by medo2132112** [comment]
Yes, are there any workarounds for this problem? Both are amazing plugins
